cucoslib/workers/gh_metadata_result_collector.py

```diff
@@ -0,0 +1,32 @@
+from selinon import StoragePool
+from cucoslib.base import BaseTask
+from cucoslib.utils import get_analysis_by_id
+
+
+class GitHubManifestMetadataResultCollector(BaseTask):
+    """
+    Collect all results that were computed, upload them to S3 and store version reference to results in WorkerResult
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def run(self, arguments):
+        self._strict_assert(arguments.get('ecosystem'))
+        self._strict_assert(arguments.get('repo_name'))
```

Comment:
`repo_name` should be just `<username>/<repo>` portion of the URL. I later store results in S3 under `username:repo/<result>.json`. I could pass just URL and then extract `repo_path` from there, when needed. But it was easier this way :)